### PR TITLE
Refactored Feeders using SPNL's TaskData

### PR DESCRIPTION
**Pull request by clementgarnier**

### CODE REVIEW

File: mry-core/src/it/scala/com/wajam/mry/storage/mysql/TestTableAllLatestFeeder.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ class TestTableAllLatestFeeder extends TestMysqlBase {
     // 91 = 100 - (5 loadMore + 4 end of records)
     records.size should be(91)
 
-    val strKeys = records.map(_(Keys).asInstanceOf[Seq[String]](0))
+    val strKeys = records.map(_.fields(Keys).asInstanceOf[Seq[String]](0))
```
Comment (alexbergeron) [inline mry-core/src/it/scala/com/wajam/mry/storage/mysql/TestTableAllLatestFeeder.scala]: `(0)` could be replaced by `.head`

**Comment by alexbergeron** [comment]
+1, but I would love @mlaflamm to give it a quick check in case I might miss some bad usage of TaskData

File: mry-core/src/it/scala/com/wajam/mry/storage/mysql/TestTableAllLatestFeeder.scala (hunk 1 above)
Comment (mlaflamm) [inline mry-core/src/it/scala/com/wajam/mry/storage/mysql/TestTableAllLatestFeeder.scala]: Agree

**Comment by mlaflamm** [comment]
+1

**Pull request merged**